<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoundingWidget</class>
 <widget class="QWidget" name="RoundingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>라운딩</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;
color: #000000;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>라운딩</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rounding_method_layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="rounding_method_label">
       <property name="styleSheet">
        <string notr="true">background: white;
color: #000000;
</string>
       </property>
       <property name="text">
        <string>라운딩 방식</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rounding_method_combo">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>반올림</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>올림</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>내림</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="decimal_places_layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="decimal_places_label">
       <property name="styleSheet">
        <string notr="true">background: white;
color: #000000;
</string>
       </property>
       <property name="text">
        <string>라운딩 자리</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="decimal_places_edit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="example_orignal_text">
       <property name="styleSheet">
        <string notr="true">background: white;
color: #000000;
</string>
       </property>
       <property name="text">
        <string>123456.7890   →</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="example_label">
       <property name="styleSheet">
        <string notr="true">background: white;
margin-left:0;
color: red;</string>
       </property>
       <property name="text">
        <string> 123456.0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="columns_table">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: #495057;
    background-color: #e9ecef;
    border: 1px solid #ced4da;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dee2e6;
}
QPushButton#save_button {
    color: #fff;
    background-color: #007bff;
    border-color: #007bff;
    font-weight: bold;
}
QPushButton#save_button:hover {
    color: #fff;
    background-color: #0069d9;
    border-color: #0062cc;
}
QPushButton#save_button:pressed {
    background-color: #0062cc;
    border-color: #005cbf;
}</string>
       </property>
       <property name="text">
        <string>저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: #495057;
    background-color: #e9ecef;
    border: 1px solid #ced4da;
    padding: 8px;
    border-radius: 4px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #dee2e6;
}</string>
       </property>
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
